hack/get-next-version.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+type flags struct {
+	CurrentVersion   string
+	ReleaseNotesFile string
+	HelmChart        bool
+}
+
+var defaultValues = &flags{
+	CurrentVersion:   "v0.0.0",
+	ReleaseNotesFile: "./release-notes.json",
+	HelmChart:        false,
+}
+
+func main() {
+	var flagValues flags
+	flag.StringVar(&flagValues.CurrentVersion, "current-version", defaultValues.CurrentVersion, "Path to the file containing the version")
+	flag.StringVar(&flagValues.ReleaseNotesFile, "release-notes-file", defaultValues.ReleaseNotesFile, "Path to the file containing the release notes in JSON format")
+	flag.Parse()
+
+	releaseNotesBytes, err := os.ReadFile(flagValues.ReleaseNotesFile)
+	if err != nil {
+		panic(fmt.Sprintf("Couldn't read '%s'", flagValues.ReleaseNotesFile))
```

Comment:
Be more explicit in the error message, and you should include the err message as well:

```suggestion
		panic(fmt.Sprintf("Failed to read release notes file '%s', error: %s", flagValues.ReleaseNotesFile, err))
```